includes/polling/os/ifotec.inc.php

```diff
@@ -0,0 +1,47 @@
+<?php
```

Comment:
This needs to be ported to the new OS format, see https://github.com/librenms/librenms/blob/master/LibreNMS/OS/Qnap.php for example.

Reply:
I followed librenms docs : https://docs.librenms.org/Developing/os/Initial-Detection/
... It's a little late to make the changes T_T
Is it mandatory ?

Reply:
Yeah docs aren't really updated.  
We want new integrations in this format yes

Reply:
I fixed the problem but I don't understand where is the problem with "Travis CI " for : 
`'hardware' => 'CPEFibre-G4G-PVN-001'`

"CPEFibre-G4G-PVN-001" isn't the product I used to create unit tests, I haven't this product in my network and I don't found "CPEFibre-G4G-PVN-001" in my source code... it is very strange...
... I don't know where "CPEFibre-G4G-PVN-001" come from

Reply:
Because it's in the current ifotec.snmprec. Please use -v (described in docs) when you add a new device with same OS.

Reply:
Ok, it is fixed with my last push